contribute/en/Release-Process.md

```diff
@@ -269,11 +269,14 @@ Approximate release date: <!-- E.g. one week after targeted translation completi
 Current state: <!-- Planning|Translations (beta)|Code freeze (rc)|Released -->
 
 **Checklist**
-- [ ] Assign this issue to the release shepherd who is in charge of managing this checklist.
+- [ ] Agree the set of changes to be included in the release.
```

Comment:
This should be formulated in a more actionable way. E.g. Where to agree? Doing that in the release issue would make sense.

Reply:
"Where" is on the project tracking board by assigning the milestone.

Reply:
Makes sense.